src/app/public/modules/toast/fixtures/toast.component.fixture.ts

```diff
@@ -20,6 +20,8 @@ import {
 export class SkyToastTestComponent {
   public toastType: SkyToastType;
 
+  public autoClose: boolean;
```

Comment:
Code style > properties should be in alphabetical order.

Reply:
Fixed.